core/primitives/src/bandwidth_scheduler.rs

```diff
@@ -614,7 +619,7 @@ mod tests {
                 .collect();
 
             let request = BandwidthRequest::make_from_receipt_sizes(
-                0,
+                ShardId::new(0),
```

Comment:
I'm trying to deprecate the use of ShardId::new in order to make sure our codebase works with arbitrary shard ids. If there is any shard layout available in the test can you use that, otherwise can use use ShardUId::single_shard to obtain the shard id?

Reply:
Ok, changed to `ShardUId::single_shard().shard_id()`.
It would be nice to have `ShardId::single_shard`, but I see that I can't use `ShardUId::single_shard()` in the `ShardId` struct because of crate dependencies :thinking: